File: Models/Dashboard/StatisticsModel.cs
namespace champi.Models.Dashboard
{
    public class StatisticsModel
    {
        public int UserCount { get; set; }
        public int TeamCount { get; set; }
        public int GameTypeCount { get; set; }
        public int LeagueCount { get; set; }
        public ChartModel LeaguePerMonthData { get; set; }
        public ChartModel LeagueByLeagueTypeData { get; set; }
    }
}

File: ClientApp/src/app/models/dashboard/statistics.model.ts
import { ChartModel } from './chart.model';

export class StatisticsModel {
    userCount: number;
    teamCount: number;
    gameTypeCount: number;
    leagueCount: number;
    leaguePerMonthData: ChartModel = new ChartModel();
    leagueByLeagueTypeData: ChartModel = new ChartModel();
}
